mage/edge_cluster.go

```diff
@@ -477,7 +478,19 @@ func genKubeconfigEntry() error {
 
 	replaceEdgeServer := string(out)
 	insideClusterUrl := "http://cluster-connect-gateway.orch-cluster.svc:8080"
-	outsideClusterUrl := "https://connect-gateway.kind.internal:443"
+
+	domainname := ""
+	// if auto-cert retrieve domainname from system
+	if autoCert {
+		// retrieve the subdomain name
+		domainname = os.Getenv("ORCH_DOMAIN")
+		if domainname == "" {
+			return fmt.Errorf("ORCH_DOMAIN is required to be set with AUTO_CERT enabled")
+		}
+	} else if domainname == "" {
+		domainname = defaultClusterDomain
+	}
+	outsideClusterUrl := fmt.Sprintf("https://connect-gateway.%s:443", domainname)
```

Comment:
Does the "serviceDomain" variable already contain the proper value?  I.e., we just need this?

outsideClusterUrl := fmt.Sprintf("https://connect-gateway.%s:443", serviceDomain)

Reply:
I think you may be right. Let me check this..